internal/dnfjson/dnfjson.go

```diff
@@ -538,6 +539,9 @@ type transactionArgs struct {
 
 	// IDs of repositories to use for this depsolve
 	RepoIDs []string `json:"repo-ids"`
+
+	// If we want weak deps for this depsolve
+	WithWeakDeps bool `json:"with_weak_deps"`
```

Comment:
I wonder if we should call this `InstallWeakDeps` to match the dnf configuration name. :thinking:

Reply:
Done :)